<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>circleRedactor</class>
 <widget class="QWidget" name="circleRedactor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>243</width>
    <height>353</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="redactorLabel">
     <property name="text">
      <string>circle redactor</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="nameLabel">
       <property name="text">
        <string>name</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="originalityLabel">
       <property name="text">
        <string>originality</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="xLabel">
       <property name="text">
        <string>x</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="yLabel">
       <property name="text">
        <string>y</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="radiusLabel">
       <property name="text">
        <string>radius</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="densityLabel">
       <property name="text">
        <string>density</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="nameLine"/>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="originalityBox"/>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="xBox"/>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="yBox"/>
     </item>
     <item row="4" column="1">
      <widget class="QSpinBox" name="radiusBox"/>
     </item>
     <item row="5" column="1">
      <widget class="QSpinBox" name="densityBox"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="setButton">
     <property name="text">
      <string>set</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
